src/styles/blocks/card.scss

```diff
@@ -0,0 +1,99 @@
+.card {
+  width: 200px;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  padding: 32px 16px 16px;
+  transition: transform ease .3s;
```

Comment:
It's a good practice to set a initial state before the transition. You should add `transform: scale(1);` to the `.card` selector.